server/src/main/java/io/crate/statistics/ReservoirSampler.java

```diff
@@ -137,6 +137,9 @@ Samples getSamples(RelationName relationName, List<Reference> columns) {
         try {
             table = schemas.getTableInfo(relationName);
         } catch (RelationUnknown e) {
+            if (LOGGER.isWarnEnabled()) {
+                LOGGER.warn("ANALYZE couldn't find table {}", relationName, e);
+            }
```

Comment:
That's the only place where we return empty stats, so wanted to get more insights if we are hitting this.
I haven't seen `Start collecting samples for table...` which is called a bit later, and afaik this is the only place where we can short circuit due to an error.

If sampling code throws elsewhere further down the line, it's visible on client - I checked with dummy exception. 

So far I haven't see any exception in the logs for past 2 days, so suspecting either short-circuit case or stuck existing run